### ISSUE TITLE
latest version of eslint-config-airbnb is not compatible with eslint 7.3.0

### ISSUE THREAD

**Issue by nescalante** [airbnb/javascript#2245]
when using latest version of eslint with latest version of eslint-config-airbnb is throwing the following error:

```
Oops! Something went wrong! :(

ESLint: 7.3.0

Error: .eslintrc.yml » eslint-config-airbnb » /blablabla/node_modules/eslint-config-airbnb-base/index.js » /blablabla/node_modules/eslint-config-airbnb-base/rules/imports.js:
	Configuration for rule "import/no-cycle" is invalid:
	Value null should be integer.
```

It seems that when parsing the rule configuration, eslint now receives:

```js
[ 'error', { maxDepth: null, ignoreExternal: false } ]
```

Removing `maxDepth` from [this line](https://github.com/airbnb/javascript/blob/54955410ee53fd82e6caa5879ccd0c413640706a/packages/eslint-config-airbnb-base/rules/imports.js#L236) fixes the issue.

**Comment by brandly** [comment]
yes! here's [a build failing as you describe](https://github.com/brandly/react-tetris/runs/788965571)

**Comment by Methuselah96** [comment]
I wonder if it has to do with the fact that JSON stringifying `Inifinity` becomes `null`?

**Comment by ljharb** [comment]
Looks like this is due to an eslint bug: https://github.com/eslint/eslint/issues/13427

I'd suggest downgrading to eslint v7.2 for now.

**Comment by brandly** [comment]
[v7.2 works as expected](https://github.com/brandly/react-tetris/runs/789429956). thanks!

**Comment by j0k3r** [comment]
Fixed by ESLint 7.3.1 https://github.com/eslint/eslint/pull/13435

**Comment by Knogobert** [comment]
They reverted this fix in the new [eslint@7.4.0](https://eslint.org/blog/2020/07/eslint-v7.4.0-released), until they find a nicer solution. So if you get a similar error, try running it in ESLint 7.3.1 for now.

`npm i eslint@7.3.1`

**Comment by taion** [comment]
No, they reverted the original config cloning change, not the band-aid hack in v7.3.1. v7.4.0 restores the pre-v7.3.0 behavior, so things should work just fine here.
